credential/builder.go

```diff
@@ -31,14 +32,15 @@ type VerifiableCredentialBuilder struct {
 }
 
 // NewVerifiableCredentialBuilder returns an initialized credential builder with some default fields populated
+// Default id is empty
 func NewVerifiableCredentialBuilder() VerifiableCredentialBuilder {
 	contexts := []string{VerifiableCredentialsLinkedDataContext}
 	types := []string{VerifiableCredentialType}
 	return VerifiableCredentialBuilder{
 		contexts: contexts,
 		types:    types,
 		VerifiableCredential: &VerifiableCredential{
-			ID:           uuid.NewString(),
+			ID:           "",
```

Comment:
🤔 given that this is a required property I'm not sure it's a good idea to leave this empty

perhaps it should be added as an optional property to the builder, and if that is empty we generate it automatically?

Reply:
yeah, it's better to keep an option argument. I'll make the changes, thanks!

Reply:
Hi @decentralgabe, I have added an optional argument to keep ID empty. Kindly review.